moduleroot/Gemfile.erb

```diff
@@ -6,7 +6,7 @@ source 'https://rubygems.org'
 gem 'puppet', ENV.key?('PUPPET_VERSION') ? "~> #{ENV['PUPPET_VERSION']}" : '>= 4.6'
 
 <% (@configs['required'] + (@configs['extra'] || [])).each do |gem| -%>
-gem '<%= gem['gem'] %>'<%= ", '#{gem['version']}'" if gem['version'] %><%= ", #{gem['options'].inspect}" if gem['options'] %>
+gem '<%= gem['gem'] %>'<%= ", '#{gem['version']}'" if gem['version'] %><%= ", #{gem['options'].inspect}" if gem['options'] %><%= " if RUBY_VERSION #{gem['ruby']['operator']} '#{gem['ruby']['version']}'" if (gem['ruby'] && gem['ruby']['operator'] && gem['ruby']['version']) %>
```

Comment:
Isn't the result here that `2.2.2` isn't quoted?

Reply:
because it's yaml, it was quoted, AFAICT